src/models/FilterGrower.js

```diff
@@ -16,6 +16,10 @@ export default class Filter {
       where.person_id = this.personId;
     }
 
+    if (this.wallet) {
```

Comment:
Need to add `wallet` to `countAppliedFilters()` below

Reply:
Feel free to refactor `countAppliedFilters()` so we don't have to do this for each new filter 😄 (but no obligation to do so).